DigitalExcellence/dex-frontend
# Feature/365 redesign project overview

## Description

Implement the new design of the project overview page, including a grid view.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the project overview page
2. There should be a button on the top right to toggle between the grid and list view
3. Both of the views should be displayed with the correct info and be responsive

## Link to issue
#365 

Closes: #365
